src/client/components/app/Transactions/Labs/LabWithdrawTx.tsx

```diff
@@ -60,11 +60,11 @@ export const LabWithdrawTx: FC<LabWithdrawTxProps> = ({ onClose, children, ...pr
   const expectedTxOutcomeStatus = useAppSelector(VaultsSelectors.selectExpectedTxOutcomeStatus);
   const actionsStatus = useAppSelector(LabsSelectors.selectSelectedLabActionsStatusMap);
 
-  const yvTokenAmount = calculateSharesAmount({
-    amount: toBN(debouncedAmount),
-    decimals: selectedLab!.decimals,
-    pricePerShare: selectedLab!.pricePerShare,
-  });
+  // NOTE: We contemplate that in labs withdraw user always will be using yvToken instead of
```

Comment:
is this correct for al labs or just specific vaults shown in labs?

Reply:
For now is correct for all labs since user uses/manipulates yvTokens. Also currently only lab with withdraw is yvBoost.